tests/authServer.test.js

```diff
@@ -0,0 +1,44 @@
+const dotenv = require('dotenv');
+dotenv.config();
+
+const authServerURL = process.env.BASE_URL + ':' + process.env.AUTH_SERVER_PORT;
+
+test('givenValidData_whenLogin_then200AndValidToken', async () => {
```

Comment:
No, I've used this naning convention on my TFG and in some other assignments. I like the explicity of it, because you don't have to search in the code to know what is being tested.